pywincffi/kernel32/file.py

```diff
@@ -131,66 +125,53 @@ def WriteFile(hFile, lpBuffer, nNumberOfBytesToWrite=None, lpOverlapped=None,
     :param handle hFile:
         The handle to write to
 
-    :type lpBuffer: bytes, string or unicode.
+    :type lpBuffer: str/bytes
     :param lpBuffer:
-        The data to be written to the file or device. We should be able
-        to convert this value to unicode.
+        Type is str on Python 2, bytes on Python 3.
```

Comment:
I'd rephrase so it's clear the doc string is talking about the objects:

```
Type is ``unicode`` on Python 2, ``str`` on Python 3.
```

You could also do:

```
Type is :class:`unicode` on Python 2, :class:`str` on Python 3.
```

I'd suggest the first version though because it render identically in Python 2 and 3 (docs could be built in either version).